### PR TITLE
WIP: Add 7in5bc support, #26

### PR DESCRIPTION
**Pull request by marcelbuesing**
I'm working on 7in5bc support for aer and since I wanted to try out the colors I added the 7in5bc version. It's mostly copy and paste besides docs and the `fn update_color_frame`. It looks like this is a little different to e.g. the 2in9bc where the update of chromatic and monochromatic part appears to happen separately. 
I still added `update_achromatic_frame` and `update_chromatic_frame` but it adds little value since it's not possible to set e.g. the chromatic part first and then only update the achromatic part (haven't tested if it works the other way around).

image

### CODE REVIEW

**Comment by caemor** [comment]
> I still added update_achromatic_frame and update_chromatic_frame but it adds little value since it's not possible to set e.g. the chromatic part first and then only update the achromatic part (haven't tested if it works the other way around).

Could you please add this to the documentation of this display and to the these two functions. 
This might be useful for people with smaller RAMs to first update the achromatic and then reuse the same buffer for the chromatic part.

Everything else looks really good

**Review by caemor — CHANGES_REQUESTED** [review]
See last message and could you maybe rebase this to main?

File: src/epd7in5bc/mod.rs [hunk 1]
```diff
@@ -0,0 +1,430 @@
...
+    DC: OutputPin,
+    RST: OutputPin,
+{
+    fn init<DELAY: DelayMs<u8>>(
+        &mut self,
+        spi: &mut SPI,
+        delay: &mut DELAY,
+    ) -> Result<(), SPI::Error> {
+        // Reset the device
+        self.interface.reset(delay, 10);
```
Comment (marcelbuesing) [inline src/epd7in5bc/mod.rs:114]: Assuming 10 is fine here but I can not verify this due to missing display.

**Comment by marcelbuesing** [comment]
> See last message and could you maybe rebase this to main?

Sorry my 7in5c display died some time last year when I accidentally dropped it and it did not make sense to order the v1 version again. I rebased the branch but it would be good to try the chromatic/achromatic fns as maybe I just tried it the wrong way around last year.

**Pull request merged**